cli/package.json

```
@@ -1,10 +1,10 @@
 {
   "name": "@epfml/disco-cli",
   "private": true,
-  "main": "dist/benchmark.ts",
+  "main": "dist/cli.ts",
```

haa oupsi, this never worked, as "dist" contains only javascript files

```suggestion
  "main": "dist/cli.js",
```